tools/circleci/Dockerfile.template

```diff
@@ -27,20 +27,22 @@
 #   3. update .circleci/config.yml to refer to the new name
 
 # TODO add xenial
-FROM buildpack-deps:trusty-scm
+FROM {base_image}
 
 RUN echo 'APT::Get::Assume-Yes "true";' > /etc/apt/apt.conf.d/90circleci \
   && echo 'DPkg::Options "--force-confnew";' >> /etc/apt/apt.conf.d/90circleci
 ENV DEBIAN_FRONTEND=noninteractive
 
 RUN apt-get update \
   && apt-get install -y \
+    sudo \
+    locales \
     xvfb \
     parallel \
     netcat unzip zip jq \
     python3-pip \
   && ln -sf /usr/share/zoneinfo/Etc/UTC /etc/localtime \
-  && { locale-gen en_US.UTF-8 || true; } \
+  && (locale-gen en_US.UTF-8 || true) \
```

Comment:
This change isn't necessary -- see the Python docs:

> Format strings contain “replacement fields” surrounded by curly braces `{}`. Anything that is not contained in braces is considered literal text, which is copied unchanged to the output. If you need to include a brace character in the literal text, it can be escaped by doubling: `{{` and `}}`.

Let's not let our templating language limit what features of our shell we can use. :)